src/odemis/gui/cont/project.py

```diff
@@ -291,6 +297,7 @@ def __init__(self, name, roc, colour, tab_data, project_panel, view_ctrl):
         self.overlay = canvas.add_roa_overlay(self.model.coordinates, colour)
         self.overlay.active.subscribe(self._on_overlay_active)
 
+    # FIXME how taken care of that his is run in the main gui thread?
```

Comment:
Add a note in the docstring "Should be called from the main GUI thread."  (and then make sure that the callers are running in the main GUI thread.